apps/site/pages/en/learn/typescript/run-natively.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Running TypeScript with a Node.js Itself
+layout: learn
+authors: AugustinMauroy
+---
+
+> **⚠️WARNING:** All content in this article uses Node.js experimental features. Please make sure you are using a version of Node.js that supports the features mentioned in this article. And remember that experimental features can change on future versions of Node.js.
+
+# Running TypeScript with a Node.js Itself
+
+In the previous articles, we learned how to run TypeScript code using transpilation and with a runner. In this article, we will learn how to run TypeScript code using Node.js itself.
+
+## Running TypeScript code with Node.js
+
+Node.js has experimental support for TypeScript. You can run TypeScript code directly in Node.js without the need to compile it first.
```

Comment:
That's not correct, it has support for a subset of TS language (or a superset of JS language) which is limited to type stripping
```suggestion
Node.js has experimental support for some TypeScript syntax. You can write code that's valid TypeScript directly in Node.js without the need to transpile it first.
```